src/__tests__/components/__snapshots__/DataDictionary.spec.tsx.snap

```diff
@@ -1,88 +1,84 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders correctly 1`] = `
+.c3 {
```

Comment:
i just learned about this and it would be useful here: you can make github hide changes to the .spec files since no one is ever going to actually review them: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/customizing-how-changed-files-appear-on-github